warehouse/models/staging/payments/elavon/stg_elavon__transactions.sql

```diff
@@ -1,108 +1,82 @@
-WITH primary_batch_transactions AS (
+{{ config(materialized='table') }}
+
+WITH
+
+source AS (
+    SELECT * FROM {{ source('elavon_external_tables', 'transactions') }}
```

Comment:
I would put the `QUALIFY DENSE_RANK() OVER (ORDER BY execution_ts DESC) = 1` here (instead of in the intermediate tables) and add a comment that we pull the whole world every day so we get only the latest extract. 

We may then also want to add a custom test to confirm that all the dates with data in the external table are present in the staging table to make sure we aren't missing anything with this filter

Reply:
I guess that second part may not be possible since I don't see a filename attribute here 🤔 Ideally we would have a filename attribute for these like for LP